File: tema_pclp_c/ex2.c
#include <stdio.h>
int main()
{
    int N,f0 = 0,fi = 0,f1 = 1;
    printf("Va rog introduceti un numar:  ");
    scanf("%d", &N);
    for(int i = 0; i < N; i++) {
        fi = f1 + f0;
        printf("%d ", fi);
        f0 = f1;
        f1 = fi;
    }


    return 0;
}

File: folder teme java/tema1/src/ex2.java
import java.io.*;
import java.lang.Math;
public class ex2 {
    public static void main(String[] args) throws IOException {
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);
        System.out.println("Va rog introduceti primul numar: ");
        int a = Integer.parseInt(br.readLine());
        System.out.println("Va rog introduceti al doilea numar: ");
        int b = Integer.parseInt(br.readLine());
        System.out.println("Va rog introduceti al treilea numar: ");
        int c = Integer.parseInt(br.readLine());
        int suma = a + b + c;
        System.out.println("Suma celor 3 numere este: " + suma);
        float media = (float) suma / 3;
        System.out.println("Media celor 3 numere este: " + media);
        int maxim= Math.max(a, b);
        System.out.print("Maximul celor 3 numere este: " + Math.max(c, maxim));
    }
}
